sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -147,6 +148,7 @@ class ScienceFiles(Base):
     ingestion_date = Column(DateTime(timezone=True))
     cr = Column(Integer, nullable=True)
     crid = Column(String, nullable=True)
+    released = Column(Boolean, nullable=False)
```

Comment:
Can we add the default here and then not need to specify the default anywhere else downstream?
```suggestion
    released = Column(Boolean, nullable=False, default=False)
```

Reply:
That's a good idea!